src/components/organisms/Footer/data.ts

```diff
@@ -14,40 +15,52 @@ export const formatter = (data: GraphQLFooterFields): FooterData => ({
 		social: { items: menuItemData.formatter(data.social?.nodes ?? []) },
 	},
 	siteTitle: data.generalSettings?.title ?? 'Superstack',
-	isHome: data.node?.isFrontPage ?? false,
 });
 
-export const getData = async (uri: string): Promise<FooterData> => {
+export const getData = async (language: Locale): Promise<FooterData> => {
```

Comment:
I would prefer to have an object as parameter to the getData function.

```suggestion
export const getData = async ({language}:{language: Locale}): Promise<FooterData> => {
```

Why? because it would be simpler to add more props to the args of the function if needed

Reply:
Done !